schema.sql

```diff
@@ -14,4 +14,28 @@ weight_kg decimal
 
 /* Day 2. */
 
-ALTER TABLE animals ADD COLUMN species VARCHAR(255);
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species VARCHAR(255);
+
+/* Day 3. */
+
+CREATE TABLE owners (
+  id INTEGER,
```

Comment:
- [ ] [OPTIONAL] I suggest you `id int NOT NULL IDENTITY(1,1),` to declare your column ids. Let me breakdown this code

   - `id:` This is the name of the column.
   - `int`: This specifies the data type of the column as an integer, which can store whole numbers.
   - `NOT NULL:` This indicates that the column cannot contain null values, meaning it must always have a value.
   - `IDENTITY(1,1)`: This is an attribute applied to the column, which defines an identity column. The (1,1) specifies that the column will automatically generate unique values starting from 1 with an increment of 1 for each new row inserted into the table. In other words, it creates an auto-incrementing column where each new row inserted will have a unique and sequentially increasing value for the id column.
   The IDENTITY property is commonly used to create primary key columns that uniquely identify each row in a table. By using IDENTITY, the database system automatically assigns and manages the values for the id column, relieving the need for manual intervention when inserting data into the table.